build.sbt

```diff
@@ -0,0 +1,31 @@
+name := "sttp-openapi-generator-example"
+
+version := "0.1"
+
+scalaVersion := "2.13.2"
+
+val root = file(".")
+
+lazy val petstoreApi = project
+  .in(file("petstore-api"))
+  .settings(
+    openApiInputSpec := s"${baseDirectory.value.getPath}/petstore.yaml",
+    openApiGeneratorName := "scala-sttp",
+    openApiOutputDir := baseDirectory.value.name,
+    openApiIgnoreFileOverride := s"$root/openapi-ignore-file",
+    libraryDependencies ++= Seq(
+      "com.softwaremill.sttp.client" %% "core" % "2.2.0",
+      "com.softwaremill.sttp.client" %% "json4s" % "2.2.0",
```

Comment:
the version should be extracted to a constant. Also, I'd use circe as it's the more popular choice

Reply:
We cannot use circe, as this plugin contains openapi-generator in 4.3.1 version which doesn't contain improved sttp generator. I've created an issue to release newer version: https://github.com/OpenAPITools/sbt-openapi-generator/issues/15

Reply:
even better, the version of the openapi-generator to use could be configured as a setting of the plugin (not hard-coded)?

Reply:
I also thought about such approach but that seems to counter with the openapi convention. Other plugins like gradle and maven are released each time a new version of generator came out. Probably there is a good reason for that. There is already an issue in sbt-openapi-plugin project to setup an automatic release process based on openapi-generator's releases. (https://github.com/OpenAPITools/sbt-openapi-generator/issues/2)